### ISSUE TITLE
multisig cannot exec 

### ISSUE THREAD

**Issue by lcg635** [EOSIO/eos#5468]
cleos get table eosio.msig southexsign1 approvals
cleos get table eosio.msig southexsign2 approvals
```
{
      "proposal_name": "s.t......e4b4",
      "requested_approvals": [],
      "provided_approvals": [{
          "actor": "southexsign2",
          "permission": "active"
        },{
          "actor": "southexsign3",
          "permission": "active"
        },{
          "actor": "southexsign1",
          "permission": "active"
        }
      ]
    }
```
The multisig transaction cannot exec sometimes.

The error is: 
```
transaction authorization failed
```

**Comment by taokayan** [comment]
Can you provide more info? like the permission level detail of your proposal, command used, etc?

**Comment by lcg635** [comment]
@taokayan 
The contract account:

```
leos get account greatsouthex
permissions:
     owner     1:    1 EOS6u9Pk37mwQ3hDL9RxtatG5VQ8nqozYnyihypJdya6fPsySN4XM
        active     3:    3 greatsouthex@eosio.code, 1 southexsign1@active, 1 southexsign2@active, 1 southexsign3@active,
        putorder     1:    1 EOS64EZvJzZq6e4PtNzXkTyP2r551mBrL6YMupJJEwyuyrNqaAZeb
memory:
     quota:     10.69 MiB    used:     748.9 KiB

net bandwidth:
     delegated:     510.0000 EOS           (total staked delegated to account from others)
     used:              10.8 KiB
     available:        327.9 MiB
     limit:            327.9 MiB

cpu bandwidth:
     delegated:     510.0000 EOS           (total staked delegated to account from others)
     used:             163.7 ms
     available:        6.128 sec
     limit:            6.292 sec
```
The transaction in the multisig is to call greatsouthex.settle method.
The multisig is created and sended use rpc api. And the expiration is 30 minute.

Here is a multisig which is not executed.
```
cleos get table eosio.msig southexsign3 approvals -l 100
{
  "rows": [{
      "proposal_name": "s.t......evpb",
      "requested_approvals": [],
      "provided_approvals": [{
          "actor": "southexsign2",
          "permission": "active"
        },{
          "actor": "southexsign1",
          "permission": "active"
        },{
          "actor": "southexsign3",
          "permission": "active"
        }
      ]
    }
  ],
  "more": false
}
```

**Comment by lcg635** [comment]
some transactions dropped by some bp.
